report/schemas.api.md

```diff
@@ -1366,6 +1368,7 @@ export type PreviewMessagePayload<T extends PreviewMessageType> = T extends Prev
     error: string;
 } : T extends PreviewMessageType.EMOTE_EVENT ? {
     type: PreviewEmoteEventType;
+    payload?: any;

```

Comment:
Could we type this? Why is it `any`?

Reply:
Sure! Initially, 'll type this to `number` and update it as required.